### DISCUSSION TITLE
API Global Query Parameters: exclude fields ?

### DISCUSSION THREAD

**Discussion by vanch3d** [directus/directus#7494] [Archive]
I'm using the API to collect the list of assets added to the system. 
The API global query parameters allow me to select which fields i want in the payload, e.g. 

```
/files?fields=id,storage,filename_disk,featured_image,metadata
```

But is there a way to exclude fields from the payload ? 
for images, I'm trying to get the EXIF out (because it's very verbose) but I'm annoyed to have to put all fields EXCEPT `metadata`

**Comment by rijkvanzanten** [comment]
No, this isn't currently possible
